### PR TITLE
Fix #32234 - Add warning when inspectdb find multi-columns primary keys

### PR DESCRIPTION
**Pull request by mdamien**
Ticket: https://code.djangoproject.com/ticket/32234

### CODE REVIEW

File: django/core/management/commands/inspectdb.py [hunk 1]
```diff
@@ -6,6 +6,10 @@
 from django.db.models.constants import LOOKUP_SEP
 
 
+class InspectDBWarning(Warning):
```
Comment (jacobtylerwalls) [inline django/core/management/commands/inspectdb.py:9]: Don't see where this is used.

File: django/db/backends/base/introspection.py [hunk 2]
```diff
@@ -164,12 +164,21 @@ def get_key_columns(self, cursor, table_name):
         raise NotImplementedError('subclasses of BaseDatabaseIntrospection may require a get_key_columns() method')
 
     def get_primary_key_column(self, cursor, table_name):
+        """
+        Return the name of the primary key column for the given table.
+        """
+        columns = self.get_primary_key_colums(cursor, table_name)
```
Comment (jacobtylerwalls) [inline django/db/backends/base/introspection.py:170]: Typo in method call. We need tests to exercise the code.

**Review by jacobtylerwalls — COMMENTED** [review]
Some first steps toward a review. Chief element missing is some tests; do say if some pointers would be helpful in getting started.

**Comment by felixxm** [comment]
Closing due to inactivity.

**Pull request closed without merging**